pkg/xds/server/callbacks/dataplane_metadata_tracker.go

```diff
@@ -27,7 +29,7 @@ func (d *DataplaneMetadataTracker) Metadata(streamId int64) *xds.DataplaneMetada
 	if found {
 		return metadata
 	} else {
-		return &xds.DataplaneMetadata{}
+		return nil
 	}
```

Comment:
```suggestion
	if found {
		return metadata
	} 
	
        return nil
```

Update this to be canonical Go style